### PR TITLE
chore: [#185149976] include webflow tasks in prod deploy job

### PR DESCRIPTION
**Pull request by jphechter**
## Description

Update production deploy job to include Webflow sync tasks

### Ticket

[185149976](https://www.pivotaltracker.com/story/show/185149976)

### Notes

Created in response to conversation at team retro.

## Code author checklist

- [X] I have performed a self-review of my code
- [X] I have created and/or updated relevant documentation, if necessary
- [X] I have not used any relative imports
- [X] I have checked for and removed instances of unused config from CMS
- [X] I have pruned any instances of unused code
- [X] I have not added any markdown to labels, titles and button text in config

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -732,6 +732,14 @@ workflows:
           requires:
             - prepare-release
             - production-hold
+      - sync-fundings:
```
Comment (aloverso) [inline .circleci/config.yml:735]: should these require the release-branch filter like the other jobs above?

Comment (jphechter) [reply .circleci/config.yml:735]: I don't think so. The `requires` option needs a successful run. If the required task is filtering and won't run, filtering again is a redundancy. At least that's my understanding.

Comment (jphechter) [reply .circleci/config.yml:735]: I guess the right thing to do would be to drop the redundant `filters`.

Comment (aloverso) [reply .circleci/config.yml:720]: ah got it, sure makes sense

**Pull request merged**